iron-resizable-behavior.html

```diff
@@ -61,6 +61,20 @@
     },
 
     attached: function() {
+      // NOTE(valdrin) In CustomElements v1 with native HTMLImports, the order
+      // of imports affects the order of `attached` callbacks (see webcomponents/custom-elements#15).
+      // This might cause a child to notify parents too early (as the parent
+      // still has to be upgraded), resulting in a parent not able to keep track
+      // of the `_interestedResizables`. To solve this, we wait for the document
+      // to be done loading before firing the event.
+      if (document.readyState === 'loading') {
+        var _this = this;
+        document.addEventListener('readystatechange', function readystatechanged() {
+          document.removeEventListener('readystatechange', readystatechanged);
+          _this.isAttached && _this.attached();
+        });
+        return;
+      }
```

Comment:
I wonder if there's a better way to do this - it feels kinda sketchy. Does an element with IronResizableBehavior know what ancestor element has it in its `_interestedResizables`? If so, maybe whenever an element with IronResizableBehavior was attached, it could ask all other things with IronResizableBehavior that aren't in anyone's `_interestedResizables` to fire `iron-request-resize-notifications` again / walk up their trees looking for themselves?

Also, what if you have:
```html
<resizable-thing-a id="A1">
  <resizable-thing-b id="B">
    <resizable-thing-a id="A2"></resizable-thing-a>
  </resizable-thing-b>
</resizable-thing-a>
```
... and `resizable-thing-a` is defined / upgrades before `resizable-thing-b`, does the tree of resizable things end up as `#A1 -> [#A2, #B]` instead of the intended `#A1 -> [#B -> [#A2]]`? Or does IronResizableBehavior already handle this situation?